releases/nightly-build/compose-files/docker-compose-nexus-add-device-services.yml

```diff
@@ -0,0 +1,122 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * @author: Leonard Goodell
+#  * EdgeX Foundry, Hanoi, version "master"
+#  * added: May, 2020
+#  *******************************************************************************/
+
+# NOTE:  this Docker Compose file does not contain the security services - namely the API Gateway
+# and Secret Store
+
+version: '3.7'
+
+# all common shared environment variables defined here:
+x-common-env-variables: &common-variables
+  REGISTRY_HOST: edgex-core-consul
+  CLIENTS_COREDATA_HOST: edgex-core-data
+  CLIENTS_NOTIFICATIONS_HOST: edgex-support-notifications
+  CLIENTS_METADATA_HOST: edgex-core-metadata
+  CLIENTS_COMMAND_HOST: edgex-core-command
+
+services:
+  device-virtual:
+    image: ${REPOSITORY}/docker-device-virtual-go${ARCH}:master
```

Comment:
placing all versions ie `:master` into a `.env` will make releases easier. Update the `.env` to pin the versions and then be done

additionally allows folks (and CI) to `export CONSUL_VERSION=1.2.3` and check everything against specific version of thing X

Reply:
I think we need to leverage .env files a lot more than we are today for sure.

Reply:
I played with this and LIKE  IT! Thanks @hutchic.

Reply:
Done